sdk/ml/azure-ai-ml/tests/test_configs/flows/summary.md

```diff
@@ -1,21 +1,124 @@
-# Summary of preferred solution
+# Flow in Pipeline experience proposal
+
+## Background
+
+After customer finalized a flow in prompt-flow, they may want to integrate it with existing pipeline. So we need to provide customer a way to use flow along with pipeline job in azure-ai-ml.
+
+### Goal
+
++ customer can refer to flow as a component in pipeline job in azure-ai-ml
+  + it should include both CLI & SDK experience
++ need to leave an interface to specify component type
+  + 1p customers need to refer a flow as a parallel component for acceleration
+  + PM has used command component for public review as it's easier to understand
+
+### Non-goal
+
++ direct referral from sources other than code-first experience won't be discussed here
+  + like drag a flow to a pipeline job in UX
+  + we'll always assume that customer has a code-first flow definition
++ detailed samples for features like side input
+
+## Related Files & Concepts
+
+- `flow.dag.yaml`: flow diagram definition yaml
+- `run.yaml`: run config for a flow run, will refer to `flow.dag.yaml`
+- `flow.meta.yaml`: include meta information for a flow. We don't have this in code-first experience for now.
+- `flow.component.yaml`: include all potential gap fields between a flow and its corresponding component. We don't have this in code-first experience for now.
+
+## Proposal Summary
+
+- support directly call `load_component` from a `flow.dag.yaml`
+
+  - SDK experience:
```

Comment:
This is experience of consumption in pipeline. May add a part create it as component after this section

Reply:
you mean something like below?
```python
ml_client.components.create_or_update(flow_component, version="3")
```